migrate/sample_data.csv

```diff
@@ -1,9 +1,9 @@
-yxc,2023-09-06 17:21:24,video/youtube,https://img.youtube.com/vi/Mr7VzZNyGW4/mqdefault.jpg,SAS Shifts Apps to ScyllaDB with No Code Changes,https://www.youtube.com/watch?v=Mr7VzZNyGW4
-qwe,2023-09-06 16:18:23,video/youtube,https://img.youtube.com/vi/sA8Rqpggr30/mqdefault.jpg,ScyllaDB Cloud Terraform provider #2: Create new cluster,https://www.youtube.com/watch?v=sA8Rqpggr30
-vdd,2023-09-05 17:28:02,video/youtube,https://img.youtube.com/vi/GMK3Hpe2VDM/mqdefault.jpg,Expert Panel The Future of NoSQL Databases,https://www.youtube.com/watch?v=GMK3Hpe2VDM
-fgh,2023-09-06 17:23:24,video/youtube,https://img.youtube.com/vi/S2xmFOAUhsk/mqdefault.jpg,How Discord Migrated Trillions of Messages from Cassandra to ScyllaDB,https://www.youtube.com/watch?v=S2xmFOAUhsk
-ewq,2023-09-06 16:19:23,video/youtube,https://img.youtube.com/vi/gZE73loJLKY/mqdefault.jpg,What’s New in ScyllaDB Open Source 5.0,https://www.youtube.com/watch?v=gZE73loJLKY
+real-time-streaming,2023-09-06 17:19:23,video/youtube,https://img.youtube.com/vi/ODAj8hW-j8c/mqdefault.jpg,Building a Real-time Streaming ETL Framework Using ksqlDB and NoSQL,https://www.youtube.com/watch?v=ODAj8hW-j8c,3142
+factors-low-latency,2023-09-06 17:19:24,video/youtube,https://img.youtube.com/vi/HrXn7UF4ktY/mqdefault.jpg,5 Factors When Selecting a High Performance Low Latency Database,https://www.youtube.com/watch?v=HrXn7UF4ktY,33
+scylladb-cloud-terraform-1,2023-09-05 17:28:02,video/youtube,https://img.youtube.com/vi/dwOxkCypE7M/mqdefault.jpg,ScyllaDB Cloud Terraform provider #3: Add CIDR block to the allowlist,https://www.youtube.com/watch?v=dwOxkCypE7M,333
+kiwi-on-scylladb,2023-09-06 17:20:24,video/youtube,https://img.youtube.com/vi/REHmq9MJP3Y/mqdefault.jpg,Kiwi.com on ScyllaDB,https://www.youtube.com/watch?v=REHmq9MJP3Y,123
+scylladb-no-code-changes,2023-09-06 17:21:24,video/youtube,https://img.youtube.com/vi/Mr7VzZNyGW4/mqdefault.jpg,SAS Shifts Apps to ScyllaDB with No Code Changes,https://www.youtube.com/watch?v=Mr7VzZNyGW4,123
+scylladb-cloud-terraform-2,2023-09-06 16:18:23,video/youtube,https://img.youtube.com/vi/sA8Rqpggr30/mqdefault.jpg,ScyllaDB Cloud Terraform provider #2: Create new cluster,https://www.youtube.com/watch?v=sA8Rqpggr30,507
+future-of-nosql,2023-09-05 17:28:02,video/youtube,https://img.youtube.com/vi/GMK3Hpe2VDM/mqdefault.jpg,Expert Panel The Future of NoSQL Databases,https://www.youtube.com/watch?v=GMK3Hpe2VDM,3382
+how-discord-migrated-trillions-of-messages,2023-09-06 17:23:24,video/youtube,https://img.youtube.com/vi/S2xmFOAUhsk/mqdefault.jpg,How Discord Migrated Trillions of Messages from Cassandra to ScyllaDB,https://www.youtube.com/watch?v=S2xmFOAUhsk,1699
+whats-new-scylladb-5,2023-09-06 16:19:23,video/youtube,https://img.youtube.com/vi/gZE73loJLKY/mqdefault.jpg,What’s New in ScyllaDB Open Source 5.0,https://www.youtube.com/watch?v=gZE73loJLKY,3242
```

Comment:
you write this data manually?

Reply:
This is just a sample data. No need to be fetching from an YouTube integration.

At this specific point, I'm using this slug to prettify how the video appears in the browser URL.

Reply:
I got it.